src/citra_qt/debugger/graphics_vertex_shader.cpp

```diff
@@ -365,7 +365,7 @@ GraphicsVertexShaderWidget::GraphicsVertexShaderWidget(std::shared_ptr< Pica::De
         input_data[i]->setValidator(new QDoubleValidator(input_data[i]));
     }
 
-    breakpoint_warning = new QLabel(tr("(data only available at VertexLoaded breakpoints)"));
+    breakpoint_warning = new QLabel(tr("(data only available at RunVS breakpoints)"));
```

Comment:
Shown in UI, so change this to "(data only available at vertex shader invocation breakpoints)"